webgoat-lessons/xxe/src/main/resources/i18n/WebGoatLabels.properties

```diff
@@ -42,5 +42,5 @@ xxe.hints.content.type.xxe.2=Does the endpoint only accept json messages?
 xxe.blind.hints.1=This assignment is more complicated you need to upload the contents of a file to the attackers site (WebWolf in this case)
 xxe.blind.hints.2=In this case you cannot combine external entities in combination with internal entities.
 xxe.blind.hints.3=Use parameter entities to perform the attack, see for example: https://www.acunetix.com/blog/articles/xml-external-entity-xxe-limitations/
-xxe.blind.hints.4=An example DTD can be found here WebGoat/images/example.dtd, include this DTD in the xml comment
+xxe.blind.hints.4=An example DTD can be found http://127.0.0.1:8080/WebGoat/xxe/sampledtd, include this DTD in the xml comment
```

Comment:
This will not work when people start WG on port 8081 for example

Reply:
I will change this with <<webgoat url>>/ and <<webwolf url>>/ (hint 5) so people can understand what to fill in there.